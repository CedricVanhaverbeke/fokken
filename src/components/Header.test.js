import { render } from '@testing-library/react';
import TeacherHeader from './Header';

describe('Teacher header', () => {
  it('should render', () => {
    const { container } = render(<TeacherHeader />);
    expect(container).toMatchSnapshot();
  });
});
