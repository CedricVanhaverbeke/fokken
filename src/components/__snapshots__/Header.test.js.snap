// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teacher header should render 1`] = `
<div>
  <nav
    class="w-full text-sm relative"
  >
    <div
      class="w-full max-w-5/6 mx-auto"
    >
      <div
        class="flex items-center justify-between h-16 w-full md:w-auto"
      >
        <div
          class="flex items-center w-full md:hidden md:w-auto"
        >
          <button
            aria-expanded="false"
            aria-label="Main menu"
            class="inline-flex items-center justify-center p-2 rounded-md transition duration-150 ease-in-out"
          >
            <svg
              class="block h-6 w-6"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M4 6h16M4 12h16M4 18h16"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
            <svg
              class="hidden h-6 w-6"
              fill="none"
              stroke="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M6 18L18 6M6 6l12 12"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
          </button>
        </div>
        <div
          class="flex-1 flex items-center justify-center md:items-stretch md:justify-start"
        >
          <div
            class="h-12 py-2 flex-shrink-0"
          >
            <svg
              class="h-full"
              id="Layer_1"
              viewBox="0 0 874 238"
              xmlns="http://www.w3.org/2000/svg"
            >
              <style>
                .st0-ftrprf
                {
                fill:#22d86b
                }
                .st1-ftrprf
                {
                fill:#ed2a24
                }
              </style>
              <path
                d="M152.6 194c6.3 0 10.3.6 10.3 9.5 0 3.9-2.2 5.8-6.9 5.8h-11.7c0 7.7 1.9 9.2 6 9.2 4.7 0 8.4-.1 12.2-.7v2.2c-4.1 1.2-8 1.4-12.2 1.4-6.9 0-8.9-3.5-8.9-13.9 0-11.9 3.2-13.5 11.2-13.5zm3.4 12.5c2.4 0 4-.6 4-2.9 0-6.1-1.7-6.6-7.4-6.6-6.2 0-8.4.6-8.4 9.6l11.8-.1zM166.3 205h15.4v2.9h-15.4V205zM199.1 184h3.1l13.1 37.2h-3.4l-3.7-11.2h-15.5l-3.8 11.2h-3.5l13.7-37.2zm8.1 23.1l-6.6-18.9-6.8 18.9h13.4zM227.3 194c2.6 0 7.6.6 11.1 1.3v2.1s-6.5-.5-11.1-.5c-3.6 0-6 1-6 11 0 8.7 2.1 10.7 6 10.7 4.7 0 11.4-.7 11.4-.7v2.2c-4.1 1.2-7.2 1.4-11.4 1.5-5 0-8.9-1.9-8.9-13.8.1-11.9 3.8-13.8 8.9-13.8zM243.3 211.7c0-4.1 2.6-6.8 6.8-6.8 0 0 11.6.1 11.6 0 0 0 .1-3.3 0-3.4 0-3.2-1.6-5-5-5-3.7 0-7.7.3-12.5.8v-2.1c3.4-.9 7.6-1.5 12.6-1.5 4.2-.1 7.8 2.2 7.8 7.8v19.6h-2.1l-.8-3.2c-.1 0-6.9 3.6-11.6 3.6-3.7 0-6.8-2.4-6.8-6.7v-3.1zm6.7 6.9c4.1 0 11.6-2.7 11.6-2.7v-8.7l-11.6.7c-2.8.2-3.9 1.9-3.9 4.1v2.6c.1 3 1.9 4 3.9 4zM279.2 193.8c2.3 0 6.7.7 10 2v-14.5h2.8v39.8h-2l-.8-3c-1.3 1.1-6.5 3.3-9.9 3.3-7.4 0-9-7.5-9-13.7-.1-6.8 1.3-13.9 8.9-13.9zm0 24.8c4 0 7.2-1.3 9.9-2.5v-18c-3.5-.9-6-1.4-9.9-1.4-3.6 0-6.1 2-6.1 11.1 0 7.5 2.5 10.8 6.1 10.8zM309.5 194c6.3 0 10.3.6 10.3 9.5 0 3.9-2.2 5.8-6.9 5.8h-11.7c0 7.7 1.9 9.2 6 9.2 4.7 0 8.4-.1 12.2-.7v2.2c-4.1 1.2-8 1.4-12.2 1.4-6.9 0-8.9-3.5-8.9-13.9 0-11.9 3.2-13.5 11.2-13.5zm3.4 12.5c2.4 0 4-.6 4-2.9 0-6.1-1.7-6.6-7.4-6.6-6.2 0-8.4.6-8.4 9.6l11.8-.1zM325.5 194.2h2l.9 3.2c3-1.8 6-3.6 10.1-3.6 3.4 0 6.2 1.6 7.4 4 2.6-2.1 6.9-4 11.2-4 4.9 0 8.3 3.7 8.3 8.7v18.6h-2.9v-18.6c0-3.3-2.2-5.8-5.4-5.8-4.1 0-7.1 1-10.6 3.1.2.4.3 2.2.3 2.6v18.7h-2.9v-18.6c0-3.3-2.2-5.8-5.5-5.8-4.1 0-6.2 1-10.1 2.9v21.5h-2.9v-26.9h.1zM371.8 194.2l6.9 21.1c1.2 3.7 4 2.9 4 2.9l8-23.9h3.2L382 229.7l-3.3 5.2H377l4.7-13.7c-1.1 0-4.4-.2-6-5.6l-6.9-21.4h3zM409.4 195l4.8-.7v-5c0-2.7.2-7.9 5.7-7.9 2.2 0 5 .3 7.3 1v1.9h-6.6c-3.3 0-3.5 2.1-3.5 5.1v5h8v2.9h-8v24.1h-2.9v-24.1h-4.8V195zM441.2 194c9 0 11.6 3.8 11.6 14.1 0 9.9-2.7 13.3-11.6 13.3-9 0-11.6-3.7-11.6-13.3 0-11 3.1-14.1 11.6-14.1zm0 24.5c5.8 0 8.7-1.1 8.7-10.5 0-10.1-2.7-11.2-8.7-11.2-5.8 0-8.7.8-8.7 11.2 0 9.7 3.1 10.5 8.7 10.5zM459.1 194.2h1.9l.9 3.4c3.1-1.8 6.1-3.7 10.4-3.7 1.5 0 3 .1 3.8.4v2.8c-.8-.2-2.3-.3-3.8-.3-4.2 0-6.5 1.3-10.4 3.1v21.2H459l.1-26.9zM494.7 194.2l9.1 22.7 9.3-22.7h3.1l-11 27h-2.9l-10.8-27h3.2zM520.2 186.9c0-.6.3-1 1-1h1.9c.6 0 1 .5 1 1v2.9c0 .6-.4 1-1 1h-1.9c-.6 0-1-.4-1-1v-2.9zm.4 7.3h2.9v27h-2.9v-27zM529.1 200.6c0-4.2 1.8-6.6 7-6.6 2.6 0 10.1.4 12.7.7v2.2h-12.7c-3.6 0-4.1 1.4-4.1 3.8v2.3c0 3.3 1.6 3.4 4.1 3.4h7.5c4.2 0 6.6 2.5 6.6 6.3v2.5c0 5.5-3.3 6.3-6.6 6.3-2.5 0-11.5-.6-13.4-.8v-2.1h13.4c.8 0 3.7.2 3.7-3.4v-2.5c0-2-1-3.4-3.7-3.4h-7.5c-4.8 0-7-1.1-7-6.3v-2.4zM555.3 186.9c0-.6.3-1 1-1h1.9c.6 0 1 .5 1 1v2.9c0 .6-.4 1-1 1h-1.9c-.6 0-1-.4-1-1v-2.9zm.4 7.3h2.9v27h-2.9v-27zM576.4 194c9 0 11.6 3.8 11.6 14.1 0 9.9-2.7 13.3-11.6 13.3-9 0-11.6-3.7-11.6-13.3 0-11 3-14.1 11.6-14.1zm0 24.5c5.8 0 8.7-1.1 8.7-10.5 0-10.1-2.7-11.2-8.7-11.2-5.8 0-8.7.8-8.7 11.2 0 9.7 3 10.5 8.7 10.5zM594.2 194.2h2l.9 3.1c3-1.8 6.2-3.6 10.3-3.6 5.4 0 8.3 4.2 8.3 8.6v18.8h-2.9v-18.8c0-3.6-2.2-5.7-5.4-5.7-4.2 0-6.4 1-10.3 3v21.6h-2.9v-27zM620.8 211.7c0-4.1 2.6-6.8 6.8-6.8 0 0 11.6.1 11.6 0 0 0 .1-3.3 0-3.4 0-3.2-1.6-5-5-5-3.7 0-7.7.3-12.5.8v-2.1c3.4-.9 7.6-1.5 12.6-1.5 4.2-.1 7.8 2.2 7.8 7.8v19.6H640l-.8-3.2c-.1 0-6.9 3.6-11.6 3.6-3.7 0-6.8-2.4-6.8-6.7v-3.1zm6.8 6.9c4.1 0 11.6-2.7 11.6-2.7v-8.7l-11.6.7c-2.8.2-3.9 1.9-3.9 4.1v2.6c0 3 1.9 4 3.9 4zM648.3 194.2h1.9l.9 3.4c3.1-1.8 6.1-3.7 10.4-3.7 1.5 0 3 .1 3.8.4v2.8c-.8-.2-2.3-.3-3.8-.3-4.2 0-6.5 1.3-10.4 3.1v21.2h-2.9l.1-26.9zM669.3 186.9c0-.6.3-1 1-1h1.9c.6 0 1 .5 1 1v2.9c0 .6-.4 1-1 1h-1.9c-.6 0-1-.4-1-1v-2.9zm.4 7.3h2.9v27h-2.9v-27zM690 194c6.3 0 10.3.6 10.3 9.5 0 3.9-2.2 5.8-6.9 5.8h-11.7c0 7.7 1.9 9.2 6 9.2 4.7 0 8.4-.1 12.2-.7v2.2c-4.1 1.2-8 1.4-12.2 1.4-6.9 0-8.9-3.5-8.9-13.9 0-11.9 3.2-13.5 11.2-13.5zm3.4 12.5c2.4 0 4-.6 4-2.9 0-6.1-1.7-6.6-7.4-6.6-6.2 0-8.4.6-8.4 9.6l11.8-.1zM704.9 200.6c0-4.2 1.8-6.6 7-6.6 2.6 0 10.1.4 12.7.7v2.2h-12.7c-3.6 0-4.1 1.4-4.1 3.8v2.3c0 3.3 1.6 3.4 4.1 3.4h7.5c4.2 0 6.6 2.5 6.6 6.3v2.5c0 5.5-3.3 6.3-6.6 6.3-2.5 0-11.5-.6-13.4-.8v-2.1h13.4c.8 0 3.7.2 3.7-3.4v-2.5c0-2-1-3.4-3.7-3.4h-7.5c-4.8 0-7-1.1-7-6.3v-2.4zM212.7 110.2c-23.6-.1-47.6-.1-70.8-.1h-26.7c-7.2 0-8.8 3.7-8.8 6.8 0 2 .6 3.6 1.8 4.8 1.8 1.8 4.5 2.1 6.9 2.1h98.2c3.6 0 6.6-1.5 7.8-4 1-1.9.8-4.1-.6-6-2-2.3-5.4-3.6-7.8-3.6zM73.4 84.8c1.2-1.2 1.8-2.9 1.7-5-.2-4.3-2.9-6.4-8.1-6.4H16.4c-1.7 0-3.5.1-5.1.9-1.5.8-3.2 2.4-3.2 5.8 0 4.5 2.8 6.8 8.3 6.8h50.5c2.1 0 4.7-.2 6.5-2.1zM101.5 148.8s-18.3-.1-27.4 0c-2.6 0-4.6.2-5.9 1.7-.8.9-1.8 2.5-1.7 5.1.1 1.9 1 6.4 7.6 6.4H101.2c2.6 0 4.5-.7 5.9-2 1.2-1.2 1.8-2.8 1.8-4.8 0-4.2-2.8-6.3-7.4-6.4z"
              />
              <path
                class="st0-ftrprf"
                d="M133.2 48.6c-.8.9-1.8 2.5-1.7 5.1.1 1.9 1 6.4 7.6 6.4h27.2c2.6 0 4.5-.7 5.9-2 1.2-1.2 1.8-2.8 1.8-4.8 0-4.2-2.9-6.8-7.5-6.8-9-.1-18.2-.1-27.4 0-2.5 0-4.5.7-5.9 2.1z"
              />
              <path
                class="st1-ftrprf"
                d="M107.5 21.4h27.2c2.6 0 4.5-.6 5.9-1.8 1.2-1.1 1.8-2.5 1.8-4.3 0-3.7-2.9-6-7.5-6.1-9 0-18.2-.1-27.4 0-2.6 0-4.5.7-5.9 1.9-.8.8-1.8 2.2-1.7 4.5.1 1.8 1 5.8 7.6 5.8z"
              />
              <path
                d="M312 9.3s-127.3-.1-137.3-.1c-2 0-8.2-.5-8.3 5.5 0 1.7-.1 3 .9 4 1.7 1.7 4.6 2.7 7 2.7 13.3-.1 73.1 0 73.1 0v48.5h-6.9c-2.4 0-4.6 1.5-5.8 2.7-1 1.1-1.1 3.4-1 5.2.1 2.5 1.4 5.2 6.8 5.2h7v65.8h-12c-2.1 0-3.9 1.3-5 2.5-1 1.1-1.5 2.6-1.5 4.4.1 2.1 1 6.3 6.7 6.3h20c2.5 0 6.8-3.4 7.1-9.7V83H304c4.1-.7 4.7-4.1 4.7-6.4 0-3.7-2.4-6.7-6.6-6.7h-39.3V21.4H312c2.1 0 3.8-1.7 3.8-3.8v-4.4c.1-2.2-1.7-3.9-3.8-3.9zm-9 74zM409 15.7v-.6c0-3.2-2.6-5.8-5.8-5.8L325 9.2c-3.2 0-5.8 2.6-5.8 5.8v.6c0 3.2 2.6 5.8 5.8 5.8h35.6l.1 134.1c0 3.7 3 6.6 6.6 6.6h2.1c3.6 0 6.4-2.9 6.4-6.4l.1-134.3 27.2.1c3.3 0 5.9-2.6 5.9-5.8zM857.3 82l-65.1-70.3c-.1-.1-.2-.1-.3-.2-.8-1.1-2-1.8-3.3-2.1h.9c-.4-.1-.8-.1-1.2-.1h-1c-35.7-.2-68.8-.1-104.8-.1-3.4 0-6.8.6-6.9 4.6 0 4.1 3.3 4.6 6.8 4.6H694v130.4h-15l-39.8-57.4c5-.4 9.8-1.3 14.3-2.9 17.2-5.8 27.1-20 27.1-39 0-10.4-2.1-26.2-16.7-34.4-10.4-5.8-21.9-6.6-33.3-5.8h-68.8c-2.6-.2-5.4-.2-8.2 0h-52.7v.2c-1.2 0-2.2-.1-2.9-.1-1.1 0-2.2 0-3.1.5-1.1.6-1.6 1.6-1.6 3.1v1.6c0 1.4.5 2.7 1.5 3.7s2.3 1.5 3.7 1.5h22.3l.1 58.3h-17c-1.8 0-3.9 0-5.6.7-2 .9-3 2.7-3 5.4 0 1.8.5 3.2 1.4 4.1 1.5 1.5 3.9 1.5 6.4 1.5H520.9l.1 59.1h-32.7l-39.7-57.4c5-.4 9.8-1.3 14.3-2.9 17.2-5.8 27.1-20 27.1-39 0-10.4-1.7-26.9-16.7-34.4-10.8-5.4-22.2-5.6-33.3-5.8h-12.8c-6.4 0-11.5 5.2-11.5 11.5v128h-15.1c-6.1 0-15.1 0-15.2 6.6-.1 5.5 6.6 6.5 7.3 6.6h128.7c5.5 0 10.9-.4 13.3-3 1-1.1 1.8-5.8 1.8-6.7V91.3h16.7c28.2 0 46.4-16.6 46.4-42.3 0-12.8-4.7-23.4-13-30.4h20.1V149h-47.1c-6.5 0-10.7 2.1-10.7 6.3s4.5 6.8 10.9 6.8h47.1c.8.1 1.6.1 2.5.1 17.9-.1 35.9 0 53.8 0h38.9c4.2 0 7.7-3.4 7.7-7.7V83c1.9.1 34.4 0 35.6 0 8.8 0 10.1-2.9 10.2-6.4.2-4.9-2.9-6.4-9.8-6.8-3.5-.2-34.1 0-36 .1V21.3h73.3l58.8 63.4-53.9 59.9c-2.3 2.6-2.7 6.3-1.1 9.3 2.1 3.9 7.4 4.6 10.3 1.2l60.2-67.9c1.3-1.5 1.2-3.7-.1-5.2zM430.7 21.4h9.7c6.6 0 16.3 1.1 21.7 4.2 3.6 1.9 11.9 8.1 11.9 23.7 0 17.7-11.5 27.9-31.5 27.9h-11.7V21.4h-.1zm.1 127.4c.2-13-.1-57.2-.1-57.2h.4l39.1 57.2h-39.4zm122.4-72.1H536V21.3h15.5c19.9 0 31.7 10.4 31.7 27.7.1 17.1-11.4 27.7-30 27.7zm68.3-55.3h9.7c6.6 0 16.3 1.1 21.7 4.2 3.6 1.9 11.9 8.1 11.9 23.7 0 17.7-11.5 27.9-31.5 27.9h-11.7V21.4h-.1zm0 127.4V91.6h.4l39.3 57.2c-13.7 0-26 .1-39.7 0z"
              />
            </svg>
          </div>
          <div
            class="hidden md:flex items-center md:ml-6"
          >
            <div
              class="mr-2"
            >
              <div
                class="py-1 px-4 rounded transition-colors duration-100 hover:bg-gray-200"
              >
                <button>
                  header.navigation.pupils
                </button>
              </div>
            </div>
            <div
              class="mr-2"
            >
              <div
                class="py-1 px-4 rounded transition-colors duration-100 bg-gray-200 font-bold"
              >
                <button>
                  header.navigation.lessons
                </button>
              </div>
            </div>
            <div
              class="mr-2"
            >
              <div
                class="py-1 px-4 rounded transition-colors duration-100 hover:bg-gray-200"
              >
                <button>
                  header.navigation.exams
                </button>
              </div>
            </div>
            <div
              class="mr-2"
            >
              <div
                class="py-1 px-4 rounded transition-colors duration-100 hover:bg-gray-200"
              >
                <button>
                  header.navigation.exercises
                </button>
              </div>
            </div>
            <div
              class="mr-2"
            >
              <div
                class="py-1 px-4 rounded transition-colors duration-100 hover:bg-gray-200"
              >
                <button>
                  header.navigation.hackRoom
                </button>
              </div>
            </div>
            <div
              class="mr-2"
            >
              <div
                class="py-1 px-4 rounded transition-colors duration-100 hover:bg-gray-200"
              >
                <button>
                  header.navigation.studio
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          class="w-full md:w-auto flex flex-row-reverse"
        >
          <div
            class="flex leading-none items-center"
          >
            <div
              class="relative inline-block"
            >
              <button
                aria-expanded="false"
                class="flex-shrink-0 hover:bg-gray-200 py-1 rounded px-4 transition-colors duration-100 flex items-center cursor-pointer"
                role="button"
                type="button"
              >
                <div
                  class="flex items-center"
                >
                  <div
                    class="mr-2"
                  >
                    <div
                      class="leading-none"
                    >
                      <div
                        class="hidden md:block"
                      >
                        header.hello
                         
                        <b />
                      </div>
                      <div
                        class="h-5 w-5 block md:hidden"
                      >
                        <svg
                          class="w-full h-full"
                          fill="currentColor"
                          height="1em"
                          stroke="currentColor"
                          stroke-width="0"
                          viewBox="0 0 496 512"
                          width="1em"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M248 104c-53 0-96 43-96 96s43 96 96 96 96-43 96-96-43-96-96-96zm0 144c-26.5 0-48-21.5-48-48s21.5-48 48-48 48 21.5 48 48-21.5 48-48 48zm0-240C111 8 0 119 0 256s111 248 248 248 248-111 248-248S385 8 248 8zm0 448c-49.7 0-95.1-18.3-130.1-48.4 14.9-23 40.4-38.6 69.6-39.5 20.8 6.4 40.6 9.6 60.5 9.6s39.7-3.1 60.5-9.6c29.2 1 54.7 16.5 69.6 39.5-35 30.1-80.4 48.4-130.1 48.4zm162.7-84.1c-24.4-31.4-62.1-51.9-105.1-51.9-10.2 0-26 9.6-57.6 9.6-31.5 0-47.4-9.6-57.6-9.6-42.9 0-80.6 20.5-105.1 51.9C61.9 339.2 48 299.2 48 256c0-110.3 89.7-200 200-200s200 89.7 200 200c0 43.2-13.9 83.2-37.3 115.9z"
                          />
                        </svg>
                      </div>
                    </div>
                  </div>
                  <svg
                    class="w-3 transition duration-150"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 448 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
                    />
                  </svg>
                </div>
              </button>
            </div>
            <div
              class="relative inline-block"
            >
              <button
                aria-expanded="false"
                class="hover:bg-gray-200 py-1 rounded px-4 transition-colors duration-100 flex items-center cursor-pointer"
                role="button"
                type="button"
              >
                <div
                  class="flex items-center"
                >
                  <div
                    class="mr-2"
                  >
                    <svg
                      class="w-5 h-5"
                      fill="none"
                      height="1em"
                      stroke="currentColor"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                      viewBox="0 0 24 24"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <circle
                        cx="12"
                        cy="12"
                        r="10"
                      />
                      <line
                        x1="2"
                        x2="22"
                        y1="12"
                        y2="12"
                      />
                      <path
                        d="M12 2a15.3 15.3 0 0 1 4 10 15.3 15.3 0 0 1-4 10 15.3 15.3 0 0 1-4-10 15.3 15.3 0 0 1 4-10z"
                      />
                    </svg>
                  </div>
                  <svg
                    class="w-3 transition duration-150"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 448 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
                    />
                  </svg>
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </nav>
  <hr
    class="w-full border-gray-300"
  />
</div>
`;
